app/app.py

```
@@ -1,219 +1,87 @@
+            raise ValueError("Production environment requires SSL database connection")
 
 def create_app():
 
+    # Initialize Flask App
     app = Flask(__name__)
 
-    # SQLAlchemy Configuration
-    app.config['SQLALCHEMY_DATABASE_URI'] = os.getenv("CONNECTION_STRING")
-    app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+    # Initialize Swagger
+    swagger = Swagger(app)
```

Ruff (0.8.2)

Local variable `swagger` is assigned to but never used

Remove assignment to unused variable `swagger`

(F841)